packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActions.tsx

```diff
@@ -1,33 +0,0 @@
-import * as React from 'react';
```

Comment:
Really weird why the git history worked on the `DocumentCard` rename to base but here it just removed the file(((